src/components/modal/modal.e2e.ts

```diff
@@ -0,0 +1,11 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('fw-modal', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<fw-modal></fw-modal>');
+    const element = await page.find('fw-modal');
+    expect(element).toHaveClass('hydrated');
+  });
+});
```

Comment:
Can we capture tests for events and other edge cases props

Reply:
WIP as mentioned above. Will update it very soon 😉